### ISSUE TITLE
Fix 'imp' Warning on Python 3

### ISSUE THREAD

**Issue by adamchainz** [aws-cloudformation/cfn-python-lint#1285]
*cfn-lint version: (`cfn-lint --version`)* 0.26.2

*Description of issue.*

I see this warning on Python 3.7:

```
.../cfnlint/helpers.py:11: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
    import imp
```

On Python 3 you can use `importlib` to the same effect as the usage of `imp`.
